static/api-docs.yaml

```diff
@@ -0,0 +1,142 @@
+openapi: 3.0.0
```

Comment:
Tested this file locally, can also try in https://editor.swagger.io/

Reply:
is the swagger UI deployed along with the app after this PR?

Reply:
Yes - it will be available at `https://app/swagger-ui`